clients/src/main/java/org/apache/kafka/clients/MetadataUpdater.java

```diff
@@ -54,7 +54,7 @@
     long maybeUpdate(long now);
 
     /**
-     * If `request` is a metadata request, handles it and return `true`. Otherwise, returns `false`.
+     * Handle disconnections for metadata requests.
```

Comment:
Seems like `handleCompletedMetadataResponse` also needs to be updated.

Reply:
Fixed. Thank you for review.